tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/SeleniumTests.cs

```diff
@@ -0,0 +1,163 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Ci.CiEnvironment;
+using Datadog.Trace.Ci.Tags;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.Ci;
+using Datadog.Trace.Util;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI;
+
+[UsesVerify]
+public class SeleniumTests : TestingFrameworkEvpTest
+{
+    public SeleniumTests(ITestOutputHelper output)
+        : base("Selenium", output)
+    {
+        SetServiceName("xunit-selenium-tests");
+        SetServiceVersion("1.0.0");
+    }
+
+    [SkippableTheory]
+    [MemberData(nameof(PackageVersions.Selenium), MemberType = typeof(PackageVersions))]
+    [Trait("RunOnWindows", "True")]
+    [Trait("Category", "EndToEnd")]
+    [Trait("Category", "TestIntegrations")]
+    public async Task Injection(string packageVersion)
+    {
+        SkipOn.Platform(SkipOn.PlatformValue.Linux);
+        SkipOn.Platform(SkipOn.PlatformValue.MacOs);
+        var tests = new List<MockCIVisibilityTest>();
+        var testSuites = new List<MockCIVisibilityTestSuite>();
+        var testModules = new List<MockCIVisibilityTestModule>();
+
+        // Inject session
+        var sessionId = RandomIdGenerator.Shared.NextSpanId();
+        var sessionCommand = "test command";
+        var sessionWorkingDirectory = "C:\\selenium\\working_directory";
+        SetEnvironmentVariable(HttpHeaderNames.TraceId.Replace(".", "_").Replace("-", "_").ToUpperInvariant(), sessionId.ToString(CultureInfo.InvariantCulture));
+        SetEnvironmentVariable(HttpHeaderNames.ParentId.Replace(".", "_").Replace("-", "_").ToUpperInvariant(), sessionId.ToString(CultureInfo.InvariantCulture));
+        SetEnvironmentVariable(TestSuiteVisibilityTags.TestSessionCommandEnvironmentVariable, sessionCommand);
+        SetEnvironmentVariable(TestSuiteVisibilityTags.TestSessionWorkingDirectoryEnvironmentVariable, sessionWorkingDirectory);
+
+        const string gitRepositoryUrl = "git@github.com:DataDog/dd-trace-dotnet.git";
+        const string gitBranch = "main";
+        const string gitCommitSha = "3245605c3d1edc67226d725799ee969c71f7632b";
+        SetEnvironmentVariable(CIEnvironmentValues.Constants.DDGitRepository, gitRepositoryUrl);
+        SetEnvironmentVariable(CIEnvironmentValues.Constants.DDGitBranch, gitBranch);
+        SetEnvironmentVariable(CIEnvironmentValues.Constants.DDGitCommitSha, gitCommitSha);
+        SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
```

Comment:
We shouldn't test with debug enabled by default

Reply:
yeah, I'm still fighting CI, I'll remove it when I get the green light

Reply:
removed in [611d0b2](https://github.com/DataDog/dd-trace-dotnet/pull/5505/commits/611d0b28eb409b24dea86bd52a843446b7086fd4)